app/code/Magento/CatalogGraphQl/etc/schema.graphqls

```diff
@@ -153,6 +153,7 @@ type CustomizableDateOption implements CustomizableOptionInterface @doc(descript
 type CustomizableDateValue @doc(description: "CustomizableDateValue defines the price and sku of a product whose page contains a customized date picker.") {
     price: Float @doc(description: "The price assigned to this option.")
     price_type: PriceTypeEnum @doc(description: "FIXED, PERCENT, or DYNAMIC.")
+    type: String @doc(description: "date, date_time or time")
```

Comment:
I would suggest the ENUM type instead if we have a predefined set of values

https://graphql.org/learn/schema/#enumeration-types

Reply:
@rogyar ,thanks for review.
This case was fixed.